JS/Lesson31/onur/my-app/src/components/ListItem/index.jsx

```diff
@@ -1,9 +1,18 @@
+import { useState } from 'react';
 import './styles.css';
 
 export const ListItem = ({ id, title, handleDelete }) => {
+
+	const [completed, setCompleted] = useState(false);
+
+	const className = (!completed) ? '' : 'completed';
```

Comment:
It's probably easier for you to read the code right now but it's a very uncommon to use parenthesis around a single condition. Instead, try doing like this:
```suggestion
	const className = !completed ? '' : 'completed';
```

Of course, if you have a more complex condition that it makes sense to add parenthesis.

Reply:
Also, negation adds cognitive complexity to the code, try to avoid it as much as possible.
Final suggestion:
```suggestion
	const className = completed ? 'completed' : '';
```